operation/fhir-operation-cqf/pom.xml

```
@@ -0,0 +1,150 @@
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <shadedArtifactAttached>true</shadedArtifactAttached>
+                            <!-- <minimizeJar>true</minimizeJar> -->
+                            <transformers>
+                                <!-- https://stackoverflow.com/a/16357984/161022 -->
+                                <transformer
+                                    implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                                    <resource>META-INF/cxf/bus-extensions.txt</resource>
+                                </transformer>
+                                <!-- Merge all default configuration files 
+                                    built for Lightbend's Configuration library -->
+                                <transformer
+                                    implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                                    <resource>reference.conf</resource>
+                                </transformer>
+                            </transformers>
+                            <!-- filter out META-INF files to avoid issues 
+                                with signed jar files -->
+                            <!-- http://zhentao-li.blogspot.com/2012/06/maven-shade-plugin-invalid-signature.html -->
+                            <filters>
+                                <filter>
+                                    <artifact>*:*</artifact>
+                                    <excludes>
+                                        <exclude>META-INF/*.SF</exclude>
+                                        <exclude>META-INF/*.DSA</exclude>
+                                        <exclude>META-INF/*.RSA</exclude>
+                                    </excludes>
+                                </filter>
+                            </filters>
+
+                            <artifactSet>
+                                <includes>
+                                    <include>com.ibm.fhir:fhir-cql*</include>
+                                    <include>com.ibm.fhir:fhir-quality-measure</include>
+                                    <include>org.opencds.cqf.cql:*</include>
+                                    <include>info.cqframework:*</include>
+                                    <include>org.fhir:ucum</include>
+                                    <include>xpp3</include>
+                                    <include>com.fasterxml.jackson.module:jackson-module-jaxb-annotations</include>
+                                    <include>org.jvnet.jaxb2_commons</include>
+                                    <include>org.eclipse.persistence</include>
+                                    <include>com.google.code.findbugs</include>
```

```suggestion
                                    <include>com.google.code.findbugs</include>
```
is this actually needed?